up-linux-streamer-plugin/src/lib.rs

```diff
@@ -0,0 +1,202 @@
+#![recursion_limit = "256"]
+
+// TODO: Consider if we're ever likely to want to use this _not_ as a Zenoh plugin, in which
+//  case the config module should be made pub and we should add validation on top of setting
+//  its members
+mod config;
+
+use crate::config::HostTransport;
+use async_std::task;
```

Comment:
Zenoh should use `tokio` instead of `async_std` in the future. I'll update the plugins, but it's fine for us to keep` async_std` temporarily.

Reply:
Agreed. Going to need to switch the up-streamer and related crates over to tokio, have an open issue for it: #11 

Marking this as resolved for now